packages/mobile/locales/en-US/inviteFlow11.json

```diff
@@ -13,7 +13,7 @@
   "redeemInviteTimeout": "Redeem Invite Timed out",
   "accountSetupFailed": "Failed to create your wallet",
   "emptyInviteCode": "Invite code has no funds",
-  "transferDollarsToAccount": "Dollars transferred to your account!",
+  "transferDollarsToAccount": "Invite funds transferred to your account!",
```

Comment:
I think this is kinda awkward phrasing for a success message. Maybe get a designer's opinion?